src/htmlexporter.cpp

```diff
@@ -327,47 +327,10 @@ void HTMLExporter::exportBasket(BasketScene *basket, bool isSubBasket)
     stream << QString(
         "  </div>\n"
         "  <p class=\"credits\">%1</p>\n").arg(
-            i18n("Made with <a href=\"http://basket.kde.org/\">%1</a> %2, a KDE tool to take notes and keep information at hand.",
+            i18n("Made with <a href=\"http://launchpad.net/basket\">%1</a> %2, a kdelibs-based tool to take notes and keep information at hand.",
```

Comment:
IMHO "KDE" gives quite clear indication to the users that a project uses KDE libs/frameworks. But it's okay to do this. There are not many places where KDE is mentioned, are there? At the same time it is important to replace URLs in the template files (basket.kde.org with https://launchpad.net/basket).

Please create such pull request against kde5port branch.